meteo-notebook.ipynb

```diff
@@ -110,10 +110,27 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    directions = np.array('N NNE NE ENE E ESE SE SSE S SSW SW WSW W WNW NW NNW N'.split())
</pre><p>Are those categories "NNE NE ENE E ESE SE SSE S SSW SW WSW W WNW NW NNW N"? and wouldn't it be clearer if you used the entire words?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rawan-a-d/Air-Quality-Prediction/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Reply:
This notation is the standard for compass directions